sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -27,6 +27,7 @@
 #include "subsystems/radio_control/sbus_common.h"
 #include BOARD_CONFIG
 #include <string.h>
+#include <stdbool.h>
```

Comment:
it shouldn't be necessary

Reply:
Yes, included via elsewhere but makes executable not bigger and IDE happier with syntax highlight ;)